.github/workflows/lint-python.yml

```diff
@@ -1,21 +1,24 @@
 name: lint_python
-on: [pull_request, push]
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
This filter shouldn't be necessary; otherwise this workflow won't be run on PRs that don't target `master`.